sos/plugins/kubernetes.py

```diff
@@ -88,6 +88,13 @@ def setup(self):
                 "{} get -o json pv".format(kube_cmd)
             ])
 
+            # get a node description from ndoes (convenience)
+            nodes = self.get_command_output("%s get nodes -o name"
+                                            % self.oc_cmd)
+            for node in nodes['output'].splitlines():
```

Comment:
Adding things "for convenience" is a common red-flag in sos; strictly, it is still primarily a tool for support data collection. That means that user convenience (for the support engineers viewing the data) comes second to requirements like:

* robustness
* space consumption
* time consumption

The user whose convenience we are most concerned with is the end-customer who is having to run sos in order to get assistance from a vendor or project.

That said, I don't have a flat-out rule on this, unless there are obvious problems for other users (unreasonably large data sets, runtime etc.

So for this I think I am happy so long as there is agreement among the maintainers of the related pieces; you guys are an active bunch and these plugins and the projects they relate to are all fairly quick-moving at the moment.

So long as we aren't making burdens for Regular Joe end-users then I'm happy with these additions.